accounts/signals.py

```diff
@@ -1,40 +1,7 @@
 from django.apps import apps
-from django.contrib.auth.management import create_permissions
 
 from project import settings
 
-# def post_migrate_create_organization(sender, **kwargs):
-#    for app_config in apps.get_app_configs():
-#        create_permissions(app_config, apps=apps, verbosity=0)
-#
-#    Organization = sender.get_model("Organization")
-#    org, created = Organization.objects.get_or_create(
-#        name="MIT", url="https://lookit.mit.edu"
-#    )
-
-
-def post_migrate_create_social_app(sender, **kwargs):
```

Comment:
Will this mess with the ability to set up the db & run through migrations from scratch, or are we ok there? Will we be able to do backwards migrations / does this code belong somewhere in a specific migration?

Reply:
We should be fine - the only code that touches SocialApp model is in the osf adapter app, and that's now disconnected from the main app and just needs to be cleaned up. The fact that Travis seems to be running without dying at test DB initialization seems promising